src/schemas/21-526EZ-allclaims/schema.js

```diff
@@ -91,14 +91,31 @@ const baseAddressDef = {
 
 const vaTreatmentCenterAddressDef = (addressSchema => {
   const { type, properties } = addressSchema;
-  return Object.assign(
-    {},
-    {
-      type,
-      required: ['country'],
-      properties: _.pick(['country', 'city', 'state'], properties)
-    }
-  );
+  return {
+    type,
+    required: ['country'],
+    properties: _.pick(['country', 'city', 'state'], properties)
+  };
+})(baseAddressDef);
+
+const form0781AddressDef = (addressSchema => {
+  const ptsdAddressOmitions = [
+    'addressLine1',
+    'addressLine2',
+    'addressLine3',
+    'postalCode',
+    'zipCode',
+  ];
+  return {
```

Comment:
Compare with https://github.com/department-of-veterans-affairs/vets-website/pull/9588/files#diff-98f463a5c9f4478a5e22c14cb08dd953L359